src/Symfony/Component/Runtime/Internal/ComposerPlugin.php

```diff
@@ -83,7 +83,7 @@ public function updateAutoloadFile(): void
         }
 
         if (!$nestingLevel) {
-            $projectDir = '__'.'DIR__.'.var_export('/'.$projectDir, true);
```

Comment:
FYI this was on purpose and breaks the plugin
@lyrixx is on a PR to revert this change and add a comment on top :)

Reply:
Oh right, thanks for letting me know!